bindings/python/tools/python/ostk/astrodynamics/czml.py

```
@@ -0,0 +1,330 @@
+        instants (list[Instant]): The sample instants.
+        coordinates (list): The sample cartesian coordinates, in meters, as a list of 3-element sequences.
+        interpolation_degree (int, optional): The Lagrange interpolation degree. Defaults to 5.
+
+    Returns:
+        dict: The CZML sampled property.
+    """
+
+    epoch: Instant = instants[0]
+
+    samples: list[float] = []
+    for instant, coordinate in zip(instants, coordinates):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)